Cassandra tests fail when adding a new entity

<!--
> Please follow the issue template below for bug reports and quiries.
> Bug reports or quiries opened without any of these info will be **closed** without any explanation.
> For feature requests, label the title with [FEATURE] and explain your use case and ideas clearly below, you can remove sections which are not relevant.
> For bug reports your `.yo-rc.json` file is mandatory else we will close the issue.
-->

##### **Overview of the issue**
When adding a new entity with the jhipster:entity generator, the CQL script generated is not automatically picked by the base class of the Cassandra tests `AbstractCassandraTest`.
Those, all tests are now failing because the table is not created when starting the SpringContext.

> Caused by: com.datastax.driver.core.exceptions.InvalidQueryException: unconfigured columnfamily task


##### **Motivation for or Use Case** 

<!-- explain why this is a bug for you -->

##### **JHipster Version(s)** 
3.0.0

##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder** 
```
{
  "generator-jhipster": {
    "jhipsterVersion": "3.0.0",
    "baseName": "jhipster",
    "packageName": "com.mycompany.myapp",
    "packageFolder": "com/mycompany/myapp",
    "serverPort": "8080",
    "authenticationType": "session",
    "hibernateCache": "no",
    "clusteredHttpSession": "no",
    "websocket": "no",
    "databaseType": "cassandra",
    "devDatabaseType": "cassandra",
    "prodDatabaseType": "cassandra",
    "searchEngine": "no",
    "buildTool": "maven",
    "enableSocialSignIn": false,
    "rememberMeKey": "5f92b9d33c406219210304a1a6e519a29cc41992",
    "useSass": false,
    "applicationType": "monolith",
    "testFrameworks": [
      "gatling"
    ],
    "enableTranslation": false
  }
}
```

##### **Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory** 
```
{
    "relationships": [],
    "fields": [
        {
            "fieldName": "title",
            "fieldType": "String"
        }
    ],
    "changelogDate": "20160329193616",
    "dto": "no",
    "service": "no",
    "entityTableName": "task"
}
```

##### **Reproduce the error** 
- create a new JHipster project with Cassandra as the database
- add a new entity with `yo jhipster:entity task`
- run `./mvnw test`

##### **Suggest a Fix** 
I suggest to automatically apply all `*added_entity*.cql` scripts automatically after starting the EmbeddedCassandra in the tests.
I'll open a pull request for it.

--- a/generators/server/templates/src/test/java/package/_AbstractCassandraTest.java
+++ b/generators/server/templates/src/test/java/package/_AbstractCassandraTest.java
@@ -11,6 +11,11 @@ import org.junit.AfterClass;
 import org.junit.BeforeClass;
 
 import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 
 /**
  * Base class for starting/stopping Cassandra during tests.
@@ -18,12 +23,23 @@ import java.io.IOException;
 public class AbstractCassandraTest {
 
     @BeforeClass
-    public static void startServer() throws InterruptedException, TTransportException, ConfigurationException, IOException {
+    public static void startServer() throws InterruptedException, TTransportException, ConfigurationException, IOException, URISyntaxException  {
         EmbeddedCassandraServerHelper.startEmbeddedCassandra();
         Cluster cluster = new Cluster.Builder().addContactPoints("127.0.0.1").withPort(9142).build();
         Session session = cluster.connect();
         CQLDataLoader dataLoader = new CQLDataLoader(session);
         dataLoader.load(new ClassPathCQLDataSet("config/cql/create-tables.cql", true, "cassandra_unit_keyspace"));
+        applyScripts(dataLoader, "config/cql/", "*_added_entity_*.cql");
+    }
+
+    private static void applyScripts(CQLDataLoader dataLoader, String cqlDir, String pattern) throws IOException, URISyntaxException {
+        Path path = Paths.get(ClassLoader.getSystemResource(cqlDir).toURI());
+        try (DirectoryStream<Path> stream = Files.newDirectoryStream(path, pattern)) {
+            for (Path entry : stream) {
+                String fileName = entry.getFileName().toString();
+                dataLoader.load(new ClassPathCQLDataSet(cqlDir + fileName, false, false, "cassandra_unit_keyspace"));
+            }
+        }
     }
 
     @AfterClass

--- a/travis/scripts/02-generate-entities.sh
+++ b/travis/scripts/02-generate-entities.sh
@@ -10,9 +10,6 @@ generateEntity() {
   local entity="$1"
   if [ -a .jhipster/"$entity".json ]; then
     yo jhipster:entity "$entity" --force --no-insight
-    if [ "$JHIPSTER" == "app-cassandra" ]; then
-      cat src/main/resources/config/cql/*_added_entity_"$entity".cql >> src/main/resources/config/cql/create-tables.cql
-    fi
   fi
 }
 

